### DISCUSSION TITLE
What's the deal with "TanStack Loaders"?

### DISCUSSION THREAD

**Discussion by shorn** [TanStack/router#692] [General]
The TanStack Router doco talks about something called [TanStack Loaders](https://tanstack.com/router/v1/docs/guide/data-loading#tanstack-loaders):
 
But the link gives a 404.  There's also a "TanStack Loaders" listed on https://tanstack.com/, but the link just goes back to the TanStack home page.
 
Is this "Loaders" project a new work-in-progress (or maybe something that is being deprecated/removed?)
Doco seems to imply it's slimmer, both in terms of features and size.

Should I just ignore Loaders for the moment and stick to using Query with Router?

Related to this - is the project open to GitHub issues raised on doco that needs improvement?  If so, do you want lots of little issues or one big issue listing a bunch of stuff?  I'm open to submitting PR's as long as the [contribution guidelines](https://github.com/TanStack/router/blob/16799907e8d27166419aa74066cc4b42daad83d4/CONTRIBUTING.md) are correct in that I don't have to commit to a CLA or similar.

**Comment by tannerlinsley** [comment]
Loaders is a lighter weight alternative to Query. PRs are accepted. The smaller pr size, the better.
